protocol/topic.go

```diff
@@ -154,14 +157,16 @@ func (topic *Topic) UnmarshalJSON(data []byte) error {
 		// skip channel - not supported for policies group
 		topic.Channel = ""
 	}
+
 	topic.Criterion = TopicCriterion(elements[index])
 	index++
-	if index < len(elements) {
-		topic.Action = TopicAction(elements[index])
-	} else {
-		topic.Action = ""
+
+	if strings.HasPrefix(elements[index], "/") {
+		index++
 	}
```

Comment:
Do we really need this check? The size of `elements` will always be a fixed one and the last element will always be a "" if there's no match for it?